_posts/2024-05-20-emacs-to-neovim.markdown

```diff
@@ -6,30 +6,34 @@ date: 2024-05-20
 tags: [emacs, neovim, spacemacs, vim, vscode, vscode-neovim, vi, editor, development, programming]
 ---
 
+### Update: 2024-05-22
+
+I continue this discussion into a "Part 2" post @ ["From Emacs to Neovim (Configuration)"]({% post_url 2024-05-23-emacs-to-neovim-part-2 %}) where some concrete configuration examples are provided.
+
 # Background
 
-I have been using emacs for around 15 years with a "traditional" configuration and key bindings setup. I’ve primarily used it for Clojure/ClojureScript development. Occasionally, I have also used it for XML, HTML, JSON, CSS, C/C++, Ruby, Python, JavaScript, and Common Lisp.
+I have been using `emacs` for around 15 years with a "traditional" configuration and key bindings setup. I’ve primarily used it for Clojure/ClojureScript development. Occasionally, I have also used it for XML, HTML, JSON, CSS, C/C++, Ruby, Python, JavaScript, and Common Lisp.
 
-About 6 years ago I started to use the [spacemacs distribution](https://www.spacemacs.org). This was done after seeing several of my coworkers using it. I thought that this would be useful due to the general knowledge sharing of common configuration advice and best practices. The defaults worked well when getting started with spacemacs to replace my existing emacs setup. My previous setup was something I had configured in an ad hoc fashion over several years with no particular setup framework in mind. Notably, I did not use `evil-mode` bindings at all. The `helm` search buffer and `magit` key sequences were the closest I had to something like vim bindings (eg. mnemonic key sequences in a "non-insert" mode).
+About 6 years ago I started to use the [spacemacs distribution](https://www.spacemacs.org). This was done after seeing several of my coworkers using it. I thought that this would be useful due to the general knowledge sharing of common configuration advice and best practices. The defaults worked well when getting started with spac`emacs` to replace my existing `emacs` setup. My previous setup was something I had configured in an ad hoc fashion over several years with no particular setup framework in mind. Notably, I did not use `evil-mode` bindings at all. The `helm` search buffer and `magit` key sequences were the closest I had to something like `vim` bindings (eg. mnemonic key sequences in a "non-insert" mode).
```

Comment:
```suggestion
About 6 years ago I started to use the [spacemacs distribution](https://www.spacemacs.org). This was done after seeing several of my coworkers using it. I thought that this would be useful due to the general knowledge sharing of common configuration advice and best practices. The defaults worked well when getting started with `spacemacs` to replace my existing `emacs` setup. My previous setup was something I had configured in an ad hoc fashion over several years with no particular setup framework in mind. Notably, I did not use `evil-mode` bindings at all. The `helm` search buffer and `magit` key sequences were the closest I had to something like `vim` bindings (eg. mnemonic key sequences in a "non-insert" mode).
```